examples/nextjs-trpc/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "nextjs-trpc",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "next build",
+    "check": "next lint && tsc --noEmit",
+    "dev": "next dev --turbo",
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "preview": "next build && next start",
+    "start": "next start",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@arcjet/next": "^1.0.0-beta.2",
```

```suggestion
    "@arcjet/next": "file:../../arcjet-next",
```

For these examples we use relative packages because we're already in the repo and we use the latest files to test new versions against all examples before we do a release.